lib/linter-plus.coffee

```diff
@@ -4,29 +4,26 @@ LinterViews = require './linter-views'
 EditorLinter = require './editor-linter'
 Helpers = require './helpers'
 Commands = require './commands'
+Messages = require './messages'
+{deprecate} = require 'grim'
 
 class Linter
   constructor:(@state)  ->
-    @state ?= {}
+    @state ?= {scope: 'File'}
 
     # Public Stuff
     @lintOnFly = true # A default art value, to be immediately replaced by the observe config below
-    @views = new LinterViews @state, this # Used by editor-linter to trigger views.render
-    @commands = new Commands this
 
     # Private Stuff
     @subscriptions = new CompositeDisposable
     @emitter = new Emitter
     @editorLinters = new Map
-    @messagesProject = new Map # Values set in editor-linter and consumed in views.render
     @linters = new Set # Values are pushed here from Main::consumeLinter
 
-    @subscriptions.add atom.config.observe 'linter.showErrorInline', (showBubble) =>
-      @views.setShowBubble(showBubble)
-    @subscriptions.add atom.config.observe 'linter.showErrorPanel', (showPanel) =>
-      @views.setShowPanel(showPanel)
-    @subscriptions.add atom.config.observe 'linter.underlineIssues', (underlineIssues) =>
-      @views.setUnderlineIssues(underlineIssues)
+    @messages = new Messages(this)
+    @views = new LinterViews this
+    @commands = new Commands this
```

Comment:
Use same style please. Either have parenthesis, or don't.

Also could use `@` instead of `this` (which is more idiomatic in coffee).